internal-packages/next-dev/README.md

```diff
@@ -99,9 +99,8 @@ export async function GET(request: NextRequest) {
 When developing a next-on-pages application, this is the development workflow that we recommend:
 
 - **Develop using the standard Next.js dev server**\
-  In order to have a very fast and polished dev experience the standard dev server provided by Next.js is the best available option. So use it to quickly make changes and iterate over them, while still having access to your Cloudflare bindings thanks to the
-  `next-dev` submodule.
+  In order to have a very fast and polished dev experience the standard dev server provided by Next.js is the best available option. So use it to quickly make changes and iterate over them, while still having access to your Cloudflare bindings thanks to the `next-dev` submodule.
```

Comment:
Add a comma after the intro clause `In order to have a very fast and polished dev experience,`.